README.md

```diff
@@ -74,14 +74,24 @@ By default the tool will try to cherry-pick the single squashed/merged commit in
 
 Based on the original pull request, creates a new one containing the backporting to the target branch. Note that most of these information can be overridden with appropriate CLI options or GHA inputs.
 
-#### Default cherry-pick strategy
+#### cherry-pick strategy
 
 The default cherry-pick strategy is `recursive` with `theirs` option for automatic conflicts resolution. Therefore, by default, all commits are cherry-picked using the following git-equivalent command:
 ```bash
-$ git cherry-pick -m 1 --strategy=recursive --strategy-option=theirs <sha>
+$ git cherry-pick -x -m 1 --strategy=recursive --strategy-option=theirs <sha>
```

Comment:
I am not that sure that the `-x` option is included right now in the executed command https://github.com/kiegroup/git-backporting/blob/ee7a87f26f0ce6863c078a2cefed1f78c68f8d57/src/service/git/git-cli.ts#L116 therefore I wouldn't add it here as _git-equivalent_ command, wdyt?

Reply:
Oh, I assumed it was but did not actually check. Good catch.

Reply:
No prob, that could be another configuration option that we could add in the tool if needed

Reply:
Removed the -x 

https://github.com/kiegroup/git-backporting/compare/601862e00450e00baea0af488345d7da60b2c6f9..0ce892ff26619c1314db4577938c4d4b2bff4cc0